runtime/compiler/z/codegen/J9CodeGenerator.cpp

```diff
@@ -125,6 +125,12 @@ J9::Z::CodeGenerator::initialize()
       cg->setSupportsInlineEncodeASCII();
       }
 
+   static bool disableInlineMath_MaxMin_FD = feGetEnv("TR_disableInlineMaxMin") != NULL;
```

Comment:
This environment variable name is bit confusing. Can we use `TR_DisableInlingMaxMinFD` (I remember I advised to shorten name from  `TR_disableInlineMath_MaxMin_FD`.